.github/actions/ubuntu-test/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:7
+FROM ubuntu:latest
```

Comment:
See ticket for rationale behind this change: https://contrast.atlassian.net/browse/NODE-1824?focusedCommentId=230368